e2e-product-testing/cypress/integration/LogIn.feature

```diff
@@ -0,0 +1,10 @@
+Feature: LogIn
+
+  I want to log in
+
+  Scenario: Logging in
```

Comment:
Even tho I love Gherkin/Cucumber, I'm not sure if it makes sense adding these, because would imply following a BDD approach, and that would be a team decision which involves both BE/FE

Reply:
@alexeh 
https://vizzuality.atlassian.net/browse/MARXAN-415#icft=MARXAN-415
https://vizzuality.atlassian.net/browse/MARXAN-408#icft=MARXAN-408

We can keep it w/o Gherkin but I believe it adds a great value. 

Product-testing does not imply following BDD approach in underlying parts of the systen.

Reply:
@kgajowy Product testing as is does not imply following BDD, **using Gherkin it does**, and It's my opinion that we should be consistent.
Product/e2e tests need to be maintained, and as we go adding more features, or more use cases that we want to be covered by these test, we should all follow the same path.
As I said, I personally like this approach, but I think we should agree as a team this approach, because implies using Gherkin and Cucumber.

But this is a very personal opinion, and since I'm a junior newcomer, I'd let FE/BE people decide

Reply:
my personal opinion - product testing seems to me to be a context where Gherkin syntax can greatly help to create scenarios with consistent structure and clarity.

As long as there are no concerns from developers and scientists about using this syntax for specs of product tests, I suggest to stick to this.

@aagm @mbarrenechea @tamaramegan @Dyostiq what do you think?

This could also serve as a testbed for a possible switch to Gherking syntax in API/geoprocessing e2e tests if we wanted to consider that. I would not hold back using this in a new context such as the one started with this PR because we are not consistent across test types - even if we did not want or were not able (time, etc.) to Gherkinize e2e test, even just the new ones forgetting about the already-written ones, I believe using this for this new area of testing would add a lot of value.